### ISSUE TITLE
Import "numpy" could not be resolved

### ISSUE THREAD

**Issue by ianpan870102** [microsoft/pyright#1695]
**Describe the bug**
As titled, import "numpy" could not be resolved. Other pip3 installed packages like matplotlib could not be resolved either.

**To Reproduce**
Have pyright 1.1.126 and Python 3.8 installed. Open a Python buffer and attempt to import numpy or matplotlib (previously installed with pip3). See the red underline that reports those imports cannot be resolved.

**Expected behavior**
No errors/warnings are reported

**Screenshots or Code**

```python
import random
import numpy as np
import math
import matplotlib.pyplot as plt

```
![image](https://user-images.githubusercontent.com/36686900/112743315-f44f4900-8fc8-11eb-9641-a98289332162.png)

**VS Code extension or command-line**
I'm running pyright in Emacs, via the [lsp-pyright](https://github.com/emacs-lsp/lsp-pyright) package.

**Additional context**
n/a

**Comment by erictraut** [comment]
You need to configure pyright appropriately so it knows about your Python environment. Please refer to the documentation for [configuration](https://github.com/microsoft/pyright/blob/master/docs/configuration.md), [settings](https://github.com/microsoft/pyright/blob/master/docs/settings.md), and [import resolution](https://github.com/microsoft/pyright/blob/master/docs/import-resolution.md).

**Comment by erictraut** [comment]
If you need additional help, feel free to start a discussion topic in the "Discussion" section.
